test/e2e/network/loadbalancer.go

```diff
@@ -969,6 +972,138 @@ var _ = common.SIGDescribe("LoadBalancers", func() {
 		ginkgo.By("hitting the TCP service's LoadBalancer")
 		e2eservice.TestReachableHTTP(tcpIngressIP, svcPort, loadBalancerLagTimeout)
 	})
+
+	ginkgo.It("should be able to preserve UDP traffic when server pod cycles for a LoadBalancer service", func() {
+		// requires cloud load-balancer support
+		e2eskipper.SkipUnlessProviderIs("gce", "gke")
```

Comment:
I copied from the others , honestly, I don't know the origin of these tests and I'm afraid of breaking someone because suddenly some tests start to run

Reply:
updated to match all cloud providers